airflow/hooks/http_hook.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+import requests
+
+from airflow import utils
+from airflow.hooks.base_hook import BaseHook
+
+
+class HttpHook(BaseHook):
+
+    """
+    Interact with HTTP servers.
+    """
+
+    def __init__(
+            self, method='POST', http_conn_id='http_default'):
+        self.http_conn_id = http_conn_id
+        self.method = method
+
+    def get_conn(self, headers):
```

Comment:
headers appears required here, because I think specifying the content-type is essential for correct HTTP communication. Browsers probably use a very common default that's only valid for how the internet was ages ago. And I have no idea whether someone wants to upload some image to a binary HTTP interface or whatever. The function doesn't specifically check for the presence of Content-Type though. Do you agree it's a good idea to require the specification of Content-Type?